api/apps/api/src/modules/clone/infra/export/note-export-started.command.ts

```diff
@@ -0,0 +1,13 @@
+import { Command } from '@nestjs-architects/typed-cqrs';
+import { ResourceKind } from '@marxan/cloning/domain';
+import { ExportId, ResourceId } from '../../export';
+
+export class NoteExportStarted extends Command<void> {
```

Comment:
from the name and the handler's code I am not sure I understand what is the role of this command/handler pair 🤔

Reply:
I mean, we mark the project export as submitted. Maybe I'm just nitpicking on the name 😉

Reply:
@hotzevzl I was struggling with the name indeed.

Consider that once we request `Export`, it is already `marked` (saved) and relevant events are emitted. This one is responsible to notify other parts of the system - in this case, ApiEvents, that the request itself was submitted.

Naming it `ExportSubmitted` may be confusing with actual application code handling export request. Furthermore, this is an intention name (command), so it would be... `MarkExportAsSubmitted`?

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/686/commits/21df305ec895e6d0a752b5ef266d08c4076a3094